src/Avalonia.Base/Threading/IDispatcher.cs

```diff
@@ -26,15 +26,6 @@ public interface IDispatcher
         /// <param name="priority">The priority with which to invoke the method.</param>
         void Post(Action action, DispatcherPriority priority = default);
 
-        /// <summary>
-        /// Posts an action that will be invoked on the dispatcher thread.
-        /// </summary>
-        /// <typeparam name="T">type of argument</typeparam>
-        /// <param name="action">The method to call.</param>
-        /// <param name="arg">The argument of method to call.</param>
-        /// <param name="priority">The priority with which to invoke the method.</param>
-        void Post<T>(Action<T> action, T arg, DispatcherPriority priority = default);
```

Comment:
Why was it removed? This overload is useful for avoiding the allocation of closures.
method can be changed like this:
```csharp
void Post(Action<object> action, object arg, DispatcherPriority priority = default);
```